sunpy/net/vso/vso.py

```diff
@@ -705,13 +705,17 @@ def download(self, method, url, dw, callback, errback, *args):
 
     @staticmethod
     def by_provider(response):
+        """Returns a dictionary of provider corresponding 
```

Comment:
This should not be formatted like this, it's a docstring not code.

Something like this would be correct. In general the triple quote marks `"""` should be on their own lines.
```python
def by_provider(response):
    """
    Returns a dictionary of provider corresponding
    to records in the response. 
    """